frontend/src/store/modules/access.js

```diff
@@ -10,6 +10,12 @@ export const useAccessStore = defineStore("access", {
   }),
   getters: {
     getRoles: (state) => state.roles,
+    allowAdoptStudent: (state) => {

```

Comment:
Remove this since it's the old way of doing permissions, we should be using the hasPermissions() function that accepts the group and action type. Those are defined in @/utils/constants.js so you'll need to add those there, something like:
```
STUDENT : {
...
  adoptPENStudent: {
    allowed: [roles....]
  }
}
```